scripts/x-data-grid-pro.exports.json

```diff
@@ -10,13 +10,15 @@
   { "name": "CursorCoordinates", "kind": "Interface" },
   { "name": "daDK", "kind": "Variable" },
   { "name": "DATA_GRID_DEFAULT_SLOTS_COMPONENTS", "kind": "Variable" },
+  { "name": "DATA_GRID_PRO_DEFAULT_SLOTS_COMPONENTS", "kind": "Variable" },
```

Comment:
Do we need to export these?

Reply:
Good point, we don't need these extra exports, removed them in https://github.com/mui/mui-x/pull/6619/commits/4f488b62629ce0470605dfdc0b89770249a841d5